Objective-C/Internal/Replicator/CBLHTTPLogic.m

```diff
@@ -143,11 +143,13 @@ - (CFHTTPMessageRef) newHTTPRequest {
     // Create the CFHTTPMessage:
     CFHTTPMessageRef httpMsg;
     if (_proxyType == kCBLHTTPProxy && _useProxyCONNECT) {
-        NSURL *requestURL = [NSURL URLWithString: $sprintf(@"%@:%d", url.host, url.my_effectivePort)];
+        NSString *destination = $sprintf(@"%@:%d", url.host, url.my_effectivePort);
+        CFURLRef requestURL = CFURLCreateWithString(kCFAllocatorDefault, (__bridge CFStringRef)destination, false);
```

Comment:
I just noticed that the last parameter is false. It should be NULL. I guess objective is kind of converting false to NULL but it should be fixed here and in 3.2 branch as well.